tests/risk/test_risk_cumulative.py

```diff
@@ -38,86 +44,100 @@ def init_instance_fixtures(self):
             end_session=end_session,
             trading_calendar=self.trading_calendar,
         )
-
-        self.algo_returns_06 = factory.create_returns_from_list(
-            answer_key.ALGORITHM_RETURNS.values,
+        self.algo_returns = factory.create_returns_from_list(
+            RETURNS,
             self.sim_params
         )
-
-        self.cumulative_metrics_06 = risk.RiskMetricsCumulative(
+        self.cumulative_metrics = risk.RiskMetricsCumulative(
             self.sim_params,
             treasury_curves=self.env.treasury_curves,
             trading_calendar=self.trading_calendar,
         )
+        for dt, returns in self.algo_returns.iteritems():
+            self.cumulative_metrics.update(
+                dt,
+                returns,
+                BENCHMARK_BASE,
+                0.0
+            )
+
+    def test_algorithm_volatility(self):
+        np.testing.assert_equal(
+            len(self.algo_returns),
+            len(self.cumulative_metrics.algorithm_volatility)
+        )
+        np.testing.assert_equal(
```

Comment:
If all we want to test is that an array contains floats, the right thing to do is `assert array.dtype == np.dtype('float64')`.  Separately, is this test even worth having if it's making no assertions about the output content?  Ditto most of the rest of the tests in this suite.